src/components/CustomerQuota/AddUserModal.tsx

```diff
@@ -99,28 +106,17 @@ export const AddUserModal: FunctionComponent<AddUserModal> = ({
       );
       Vibration.vibrate(50);
       if (ids.indexOf(id) > -1) {
-        throw new Error(
-          "You've added this ID before, please scan a different ID."
-        );
+        throw new Error("Enter or scan a different ID number.");
       }
       addId(id);
       setIsVisible(false);
       setIdInput("");
     } catch (e) {
       setIsScanningEnabled(false);
-      Alert.alert(
-        "Error",
-        e.message || e,
-        [
-          {
-            text: "OK",
-            onPress: () => setIsScanningEnabled(true)
-          }
-        ],
-        {
-          onDismiss: () => setIsScanningEnabled(true) // for android outside alert clicks
-        }
-      );
+      if (e instanceof EnvVersionError) {
+        // todo: alert for env version errors
+      }
+      showAlert({ ...wrongFormatAlertProp, description: e.message });
```

Comment:
were you able to see the alert modal in the add user modal? i cant seem to see the alert